### PR TITLE
Decode protobuf Buffers as Uint8Arrays

### LINKED ISSUE #239: Wrap protobufjs `Field` to use `Uint8Array` instead of `Buffer` in Node.js

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #239

I don't see a straightforward way to wrap `Field`. This is an 80/20 solution.﻿

### CODE REVIEW

File: packages/proto-signing/src/registry.ts [hunk 1]
```diff
@@ -94,7 +94,13 @@ export class Registry {
       return this.decodeTxBody(value);
     }
     const type = this.lookupTypeWithError(typeUrl);
-    return type.decode(value);
+    const decoded = type.decode(value);
+    Object.entries(decoded).forEach(([key, val]: [string, any]) => {
+      if (val.constructor.name === "Buffer") {
```
Comment (webmaster128) [inline packages/proto-signing/src/registry.ts:99]: We can also use `Buffer.isBuffer`:

```ts
  if (typeof Buffer !== "undefined" && typeof Buffer.isBuffer !== "undefined") {
    // Buffer.isBuffer is available at runtime
    if (Buffer.isBuffer(data)) return false;
  }
```

See isUint8Array

Comment (webmaster128) [reply packages/proto-signing/src/registry.ts:99]: Alternatively we can simply do

```ts
if (Object.prototype.toString.call(val) === "[object Uint8Array]") {
  // here val is `Uint8Array` or `Buffer`
  decoded[key] = Uint8Array.from(val);
}
```

Comment (willclarktech) [reply packages/proto-signing/src/registry.ts:99]: I like the `Buffer.isBuffer` option as being the closest to what we actually mean.

**Comment by willclarktech** [comment]
Code coverage loss is due to the `Buffer` check.

**Pull request merged**